internal/cloudsql/instance_test.go

```diff
@@ -156,7 +156,7 @@ func TestConnectInfoErrors(t *testing.T) {
 	if err != nil {
 		t.Fatalf("%s", err)
 	}
-	defer cleanup()
+	defer cleanup(true)
```

Comment:
So taking a second look, it seems like we ignore the error in this situations. In that case, do we really need a true/false option or can we just ignore the received value?

We should probably pick one and be consistent - we either pass in a true/false to ignore uncalled endpoints (maybe preferable on the off chance it gets a different error some day), or just all cleanup and ignore the output.

Reply:
Sorry -- I was mindlessly typing on the last commit. Yes, let's just cleanup and ignore the error. We'll introduce an argument when we need it, but until this, a simple cleanup without an error check seems best.